packages/xo-web/src/xo-app/backup-ng/new/index.js

```diff
@@ -611,6 +635,26 @@ export default decorate([
               }
             : setting
         ),
+      displayAdvancedSettings: (
+        {
+          _displayAdvancedSettings,
+          compression,
+          isFull,
+          propSettings,
+          settings = propSettings,
+        },
+        { job }
+      ) =>
```

Comment:
Do not access to state/props you don't need to avoid unnecessary re-computations.